web/wp-content/themes/dctx-v2/inc/acf.php

```diff
@@ -0,0 +1,449 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-kzv7l2.php, but found phpcs-scan-kzV7l2.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).